lib/libyear_bundler/models/ruby.rb

```diff
@@ -102,11 +103,11 @@ def installed_version_sequence_index
       end
 
       def newest_version_sequence_index
-        all_stable_versions.index(newest_version.to_s)
+        all_stable_versions.index { |ver| ver == newest_version.to_s }
       end
 
       def release_date(version)
-        v = all_stable_versions.detect { |ver| ver['version'] == version }
+        v = all_stable_versions.detect { |ver| ver == version }
```

Comment:
Maybe instead of this line, the conditional below could be `if all_stable_versions.include?(version)`?

Reply:
We would still need to do the detection, which would add another iteration through `all_stable_versions`.